src/App.tsx

```diff
@@ -15,71 +19,96 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export class App extends Component<{}, State> {
+  state = {
+    selectedGood: 'Jam',
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  addGood = (good: string) => {
+    this.setState({ selectedGood: good });
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  clearGood = () => {
+    this.setState({ selectedGood: '' });
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  render(): React.ReactNode {
+    const { selectedGood } = this.state;
+    const { clearGood, addGood } = this;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+    return (
+      <main className="section container">
+        {
+          selectedGood === ''
+            ? <h1 className="title">No goods selected</h1>
+            : (
+              <div className="is-flex is-align-items-center">
+                <h1 className="title mb-0">{`${selectedGood} is selected`}</h1>
+                <button
+                  data-cy="ClearButton"
+                  type="button"
+                  className="delete ml-3"
+                  onClick={clearGood}
+                >
+                  {}
+                </button>
+              </div>
+            )
+        }
```

Comment:
```suggestion
        {
          selectedGood
            ? (
              <h1 className="title is-flex is-align-items-center">
                {`${selectedGood} is selected`}
                {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
                <button
                  data-cy="ClearButton"
                  type="button"
                  className="delete ml-3"
                  onClick={clearGood}
                />
              </h1>
            )
            : <h1 className="title">No goods selected</h1>
        }
```
Fix bug from video:)

https://user-images.githubusercontent.com/119679758/215580679-2b192b87-8819-4d62-a29c-48c5dbe0c0cd.MOV